### PR TITLE
Fix jdbc pool configs

### PR DESCRIPTION
**Pull request by anupama-pathirage**
Fixes #<Issue Number>

## Check List 
- [ ] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Required Balo version update
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [ ] Added necessary tests
  - [ ] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [ ] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

File: stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal [hunk 1]
```diff
@@ -36,50 +36,48 @@ public type ClientEndpointConfig record {|
...
+#                         the configuration API with the key `b7a.jdbc.pool.dataSourceClassName`.
 # + autoCommit - Auto-commit behavior of connections returned from the pool.
-#                Default value of this field can be set through the configuration API with the key
-#                "b7a.jdbc.connection.auto.commit"
-# + isXA - Whether Connections are used for a distributed transaction.
-#          Default value of this field can be set through the configuration API with the key "b7a.jdbc.xa.enabled"
+#                Default value is `true` and it can be changed through the configuration API with the key
+#                `b7a.jdbc.pool.autoCommit`.
+# + isXA - Whether Connections are used for a distributed transaction. Default value is `false` and it
+#          can be set through the configuration API with the key `7a.jdbc.pool.isXA`.
```
Comment (Manuri) [inline stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal]:
```suggestion
#          can be set through the configuration API with the key `b7a.jdbc.pool.isXA`.
```

File: stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal [hunk 2]
```diff
@@ -36,50 +36,48 @@ public type ClientEndpointConfig record {|
...
-#                 Default value of this field can be set through the configuration API with the key
-#                 "b7a.jdbc.connection.max.life.time"
-# + validationTimeout - Maximum amount of time that a connection will be tested for aliveness. Default 5 seconds
-#                       Default value of this field can be set through the configuration API with the key
-#                       "b7a.jdbc.validation.time.out"
+#                 maximumPoolSize and it can be changed through the configuration API with the key
+#                 `b7a.jdbc.pool.minimumIdle`.
+# + maxLifetimeInMillis - Maximum lifetime of a connection in the pool. Default is 1800000 (30 minutes) and it can be
+#                  changed through the configuration API with the key `b7a.jdbc.pool.maxLifetimeInMillis`. A value of 0
+#                  indicates no maximum lifetime (infinite lifetime), subject of course to the `idleTimeoutInMillis`.
```
Comment (Manuri) [inline stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal]:
```suggestion
#                  indicates unlimited maximum lifetime (infinite lifetime), subject of course to the `idleTimeoutInMillis`.
```

File: stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal [hunk 3]
```diff
@@ -36,50 +36,48 @@ public type ClientEndpointConfig record {|
...
+# + validationTimeoutInMillis - Maximum amount of time that a connection will be tested for aliveness. Default is
```
Comment (Manuri) [inline stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal]:
```suggestion
# + validationTimeoutInMillis - Maximum duration of time that a connection will be tested for aliveness. Default is
```

File: stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal [hunk 4]
```diff
@@ -36,50 +36,48 @@ public type ClientEndpointConfig record {|
...
-#                       "b7a.jdbc.connection.time.out"
-# + idleTimeout - Maximum amount of time that a connection is allowed to sit idle in the pool. Default is 10 minutes.
-#                 Default value of this field can be set through the configuration API with the key
-#                 "b7a.jdbc.connection.idle.time.out"
+#                     Default value is 15 and it can be changed through the configuration API with the key
+#                     `b7a.jdbc.pool.maximumPoolSize`.
+# + connectionTimeoutInMillis - Maximum number of milliseconds that a client will wait for a connection from the pool.
+#                       Default is 30000 (30 seconds) and it can be changed through the configuration API with the key
+#                       `b7a.jdbc.pool.connectionTimeoutInMillis`. Lowest acceptable connection timeout is 250 ms.
+# + idleTimeoutInMillis - Maximum amount of time that a connection is allowed to sit idle in the pool. Default is
```
Comment (wggihan) [inline stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal]:
```suggestion
# + idleTimeoutInMillis - The maximum amount of time that a connection is allowed to sit idle in the pool. Default is
```

File: stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal [hunk 5]
```diff
@@ -36,50 +36,48 @@ public type ClientEndpointConfig record {|
...
-#                       Default value of this field can be set through the configuration API with the key
-#                       "b7a.jdbc.connection.time.out"
-# + idleTimeout - Maximum amount of time that a connection is allowed to sit idle in the pool. Default is 10 minutes.
-#                 Default value of this field can be set through the configuration API with the key
-#                 "b7a.jdbc.connection.idle.time.out"
+#                     Default value is 15 and it can be changed through the configuration API with the key
+#                     `b7a.jdbc.pool.maximumPoolSize`.
+# + connectionTimeoutInMillis - Maximum number of milliseconds that a client will wait for a connection from the pool.
+#                       Default is 30000 (30 seconds) and it can be changed through the configuration API with the key
+#                       `b7a.jdbc.pool.connectionTimeoutInMillis`. Lowest acceptable connection timeout is 250 ms.
```
Comment (wggihan) [inline stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal]:
```suggestion
#                       `b7a.jdbc.pool.connectionTimeoutInMillis`. Lowest acceptable connection timeout value is 250 ms.
```

To match with other places.

File: stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal [hunk 6]
```diff
@@ -36,50 +36,50 @@ public type ClientEndpointConfig record {|
...
-#                 "b7a.jdbc.connection.min.idle.count"
-# + maxLifetime - Maximum lifetime of a connection in the pool. Default is 30 minutes.
-#                 Default value of this field can be set through the configuration API with the key
-#                 "b7a.jdbc.connection.max.life.time"
-# + validationTimeout - Maximum amount of time that a connection will be tested for aliveness. Default 5 seconds
-#                       Default value of this field can be set through the configuration API with the key
-#                       "b7a.jdbc.validation.time.out"
+#                Default value is `true` and it can be changed through the configuration API with the key
+#                `b7a.jdbc.pool.autoCommit`.
+# + isXA - Whether Connections are used for a distributed transaction. Default value is `false` and it
```
Comment (Manuri) [inline stdlib/jdbc/src/main/ballerina/src/java.jdbc/jdbc_types.bal]:
```suggestion
# + isXA - Whether connections are used for a distributed transaction. Default value is `false` and it
```

**Pull request merged**